pkg/util/system/version_linux.go

```diff
@@ -40,21 +26,43 @@ func GetOsVersion() (string, error) {
 	return "Unknown Linux version", fmt.Errorf("Unknown Linux version")
 }
 
-func runCommand(command string, args ...string) (string, error) {
-	e := exec.Command(command, args...)
-	data, err := e.CombinedOutput()
-	text := string(data)
-	text = strings.TrimSpace(text)
+func extractPrettyNameFromOsRelease() (string, error) {
+	file := "/etc/os-release"
+	buf, err := ioutil.ReadFile(file)
```

Comment:
If we were to call `getFileContents` here as well and we would change `getFileContents` to be an implementation of a say `ReleaseFileGetter` interface, we actually would get into a position where we could test this. 

The default implementation would still read from the file system, but for testing we can return arbitrary data. 

WDYT?

Reply:
Also, one could use the string returned by `getFileContent` and split it at '\n' and pass the slice then to `util.ExtractKeyValuePairs` which returns a map of key/value pairs. Then one just can check for the "PRETTY_NAME" key.

Reply:
@hferentschik Yeah, done.